src/olympia/promoted/tests/test_utils.py

```diff
@@ -65,7 +73,50 @@ def test_create_stripe_checkout_session():
                     args=[addon.id],
                 )
             ),
-            line_items=[{"price": "sponsored-price-id", "quantity": 1}],
+            line_items=[{"price": "yearly-sponsored-price-id", "quantity": 1}],
+            customer_email=customer_email,
+        )
+
+
+@override_settings(
+    STRIPE_API_VERIFIED_MONTHLY_PRICE_ID="monthly-verified-price-id"
+)
+def test_create_stripe_checkout_sessio_for_sponsored():
```

Comment:
I believe this test is for verified, not sponsored. Also, you're missing an `n` in the word `sessio`.